pages/clubs/Running.jsx

```diff
@@ -1,5 +1,8 @@
 var React = require('react');
 
+var config = require('../../config/config');
+const TEACH_API_URL = config.TEACH_API_URL.replace(/\/?$/, '');
```

Comment:
this should tap into the teach-api (which can be passed into component from a parent)